lib/services.js

```diff
@@ -168,6 +169,14 @@ const services = {
         // will reassign some of the above values with the ones
         // from the intiation of the multipart upload
         // (for instance, storage class)
+        const validateUserMdRes = validateUserMetadata(metaHeaders);
```

Comment:
If we add the check here will we only provide the error for an mpu on the complete step?  It should be on the initiate step.